eng/pipelines/stress/http-linux.yml

```diff
@@ -1,5 +1,12 @@
 trigger: none
 
+schedules:
+- cron: "0 5 * * *"
```

Comment:
Would it make sense to do it like at 2-3 am when machines are not busy?

Reply:
3 am is when the outerloop tests kick in, so moved it a bit later to avoid traffic. 5 am UTC is 9pm Pacific, do we expect many builds at that hour?

Reply:
You never know, sometimes people work that late, I would say it would be best to move this one to like 5am PST and the outerloop ones to 2am PST

Reply:
Looks like the intended times for outerloop were 3am PST instead of UTC. Will merge this now and follow-up with a PR that corrects outerloop triggers tomorrow.